### DISCUSSION TITLE
How to make the contents fill the page & change line spacing?

### DISCUSSION THREAD

**Discussion by MasonChan314159** [alshedivat/al-folio#704] [Q&A]
Hi! I am currently making my GitHub Page, things are going on smoothly and the theme is great, but I have no idea how to make the contents fill up the page. 

Also, I am trying to make a list, but the spacing between the items is too big (see the attachment). How to reduce the spacing for that? 

Any help is appreciated! 😄

<img width="1440" alt="Screenshot 2022-06-12 at 23 47 25" src="https://user-images.githubusercontent.com/107308887/173241679-95fd487b-f6f2-4db0-8481-ffdfcc9d9438.png">

**Comment by rohandebsarkar** [comment]
You can increase the maximum site width with the following variable in `_config.yml`

https://github.com/alshedivat/al-folio/blob/bcd43a18fe67da84193fb6df21b0f27f993fe5f8/_config.yml#L42
